### PR TITLE
Classifier: Add an id parameter to task default annotations

### PR DESCRIPTION
**Pull request by eatyourgreens**
Add an optional id parameter to `task.defaultAnnotation` and change the code throughout to use `task.defaultAnnotation()`.

_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package:
lib-classifier

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
If tasks could reflect on themselves to get their annotation model, that would save a lot of repetition. Something like:
```js
    defaultAnnotation (id = cuid()) {
      return self.annotationModel.create({
        id,
        task: self.taskKey,
        taskType: self.type
      })
    },
```

File: packages/lib-classifier/src/plugins/tasks/readme.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The [base Task model](https://github.com/zooniverse/front-end-monorepo/tree/mast
 
 All tasks should extend the Task model by implementing the following:
 
-- _defaultAnnotation (Annotation)_ the default annotation for this task.
+- _defaultAnnotation(id) (Annotation)_ return a new default annotation for this task. The `id` parameter is optional.
```
Comment (srallen) [inline packages/lib-classifier/src/plugins/tasks/readme.md:64]:
```suggestion
- _defaultAnnotation(id) (Annotation)_ return a new default annotation for this task. The `id` parameter is optional and defaults to a generated id using cuid.
```

**Review by srallen — COMMENTED** [review]
This is tentatively approved, but I'd like to manually check default annotations with the dev classifier with a few projects, but can't right now due to the subject asset migration happening on staging. I'll try again tomorrow. In the meantime, I have one minor doc update suggestion.

**Review by srallen — APPROVED** [review]
The default annotation for single choice, multiple choice, drawing, text, and transcription tasks looks good in the dev classifier. I'm presuming the simple dropdown is fine too, but I don't know a project id of a good test project for that one.

**Pull request merged**